modules/setting/setting.go

```diff
@@ -732,6 +736,14 @@ func NewContext() {
 	if len(serverCiphers) > 0 {
 		SSH.ServerCiphers = serverCiphers
 	}
+	serverKeyExchanges := sec.Key("SSH_SERVER_KEY_EXCHANGES").Strings(",")
```

Comment:
Shouldn't this be a empty string ? 
For example if SSH_SERVER_KEY_EXCHANGES is not set serverKeyExchanges = "," -> len(serverKeyExchanges) > 0 == true

Reply:
Sorry misread ...